api/openapi/v1/support-notifications.yaml

```diff
@@ -74,23 +82,23 @@ paths:
                 $ref: '#/components/schemas/Error'
   /v1/notification/age/{age}:
     delete:
-      description: Delete the proccessed Notifications if the current timestamp minus
+      description: Delete the proccessed notifications if the current timestamp minus
         their last modification timestamp is less than the age parameter, and the
-        corresponding Transmissions will also be deteled.  Please notice this API
-        is only for proccessed Notifications (status = PROCCESSED).  If the deletion
-        purpose includes each kind of Notifications, please refer to /cleanup API.
+        corresponding transmissions will also be deteled. This API is only for
+        proccessed notification, where status = PROCCESSED. If the deletion
+        purpose includes each kind of notification, please refer to the /cleanup API.
       parameters:
       - name: age
         in: path
-        description: Specify the age of Notification, and the format is in milliseconds.
+        description: Specify the age of notification, and the format is in milliseconds.
         required: true
         style: simple
         explode: false
         schema:
           type: number
       responses:
         200:
-          description: Return "True" to represent successful.
+          description: Return true to represent success.
```

Comment:
This is awkward.

Bring this in line with other Swagger HTTP 200 descriptions.